packages/commonwealth/client/scripts/views/components/component_kit/CWContentPage/CWContentPage.tsx

```diff
@@ -167,6 +180,13 @@ export const CWContentPage = ({
     </div>
   );
 
+  const disabledActionsTooltipText = (() => {
+    if (!hasJoinedCommunity) return 'Join community to upvote';
+    if (thread.archivedAt) return 'Thread is archived';
```

Comment:
`CWContentPage` is used on the snapshots and proposals pages, where there is no thread. Hence, `thread.archivedAt` will result in the following error `Cannot read properties of undefined (reading 'archivedAt')` when you click to view a specific snapshot or proposal. 

I would recommend using `thread?.archivedAt`, instead.

Reply:
Nice spot, thanks. I have fixed it [here](https://github.com/hicommonwealth/commonwealth/pull/5465/commits/b31fa70f5fdded9b7a0f6d9161e9a8a0d432ad44)